Tests/js/views/services/assets/ez-locationviewviewservice-tests.js

```diff
@@ -430,6 +430,216 @@ YUI.add('ez-locationviewviewservice-tests', function (Y) {
             Y.Mock.verify(app);
             Y.Mock.verify(contentMock);
         },
+
+        "Should launch the universal discovery widget when receiving an moveAction event": function () {
+            this.service.on('contentDiscover', function (e) {
+                Y.Assert.isObject(e.config, "contentDiscover config should be an object");
+                Y.Assert.isFunction(e.config.contentDiscoveredHandler, "config should have a function named contentDiscoveredHandler");
+            });
+            this.service.fire('whatever:moveAction');
+        },
+    });
+
+    moveContentTest = new Y.Test.Case({
+        name: "eZ Location View View Service move content tests",
+
+        setUp: function () {
+            var that = this;
+            this.capiMock = new Y.Test.Mock();
```

Comment:
side note: you can write `Y.Mock` or you can even define the shortcut at the beginning of the test to get rid of `Y.`.